src/Simulation/Simulators/QuantumExecutor/Allocate.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Quantum.Simulation.Core;
+
+namespace Microsoft.Quantum.Simulation.QuantumExecutor
+{
+    public partial class QuantumExecutorSimulator
+    {
+        public class QuantumExecutorSimAllocate : Intrinsic.Allocate
```

Comment:
It does not really matter in this case since this name is local and is not used anywhere at all. But I see your point, it is better to be clean everywhere. Fixing.

---
In reply to: [348342120](https://github.com/microsoft/qsharp-runtime/pull/82#discussion_r348342120) [](ancestors = 348342120)